main.go

```diff
@@ -105,9 +106,8 @@ func loadConfig() kubernetes.Interface {
 // main entry point of the program
 func main() {
 	var wg sync.WaitGroup
-
 	clientset := loadConfig()
-	sharedInformers := informers.NewSharedInformerFactory(clientset, viper.GetDuration("resync-interval"))
+	sharedInformers := informers.NewSharedInformerFactoryWithOptions(clientset, viper.GetDuration("resync-interval"), informers.WithNamespace(viper.GetString("WATCH_NAMESPACE")))
```

Comment:
I'm unfamiliar with the behavior but what happens if the var is missing?  The viper docs do not seem to be specific

Reply:
Hi @jcantrill, it works like it was not defined, so it keeps the previous behaviour.